Puppetfile

```diff
@@ -3,13 +3,10 @@ forge 'https://forgeapi.puppetlabs.com'
 mod 'katello/common',                '>= 1.0.1 < 1.1.0'
 mod 'katello/service_wait',          '>= 1.0.3 < 1.1.0'
 
-mod 'katello/certs',                 '>= 3.0.0 < 3.1.0'
-mod 'katello/qpid',                  '>= 3.1.0 < 3.2.0'
-mod 'katello/pulp',                  '>= 5.0.0 < 5.1.0'
-mod 'katello/candlepin',             '>= 4.0.0 < 4.1.0'
+mod 'katello/certs',                 '>= 3.0.1 < 3.0.2'
```

Comment:
Why does this pin need changing? It already allowed the update to happen.

Reply:
Typo, will revert.